rotkehlchen/globaldb/manual_price_oracles.py

```diff
@@ -51,14 +50,7 @@ def query_historical_price(
         )
 
 
-class ManualCurrentOracle(CurrentPriceOracleInterface):
-
-    def __init__(self) -> None:
-        super().__init__(oracle_name='manual current price oracle')
-
-    def rate_limited_in_last(self, seconds: Optional[int] = None) -> bool:
-        return False
-
+class ManualCurrentOracle:
```

Comment:
Why not do what we discussed in discord?

![2022-09-30_14-04](https://user-images.githubusercontent.com/1658405/193265741-3547ebd6-1d2f-42ae-b9e8-86f873f4de5c.png)

This way you will have to use `Union` for types everywhere while with inheritance you can use the base type.

Sure, at some point you will have to do a type: ignore for the function signature not agreeing with supertype for the ManualCurrentOracle but that's fine, no?